docs/storing-querying/querying-data/index.md

```diff
@@ -98,30 +94,46 @@ Each type of self-describing event is in a dedicated `OBJECT`-type column. The c
 You can query fields in the self-describing event like so:
 
 ```sql
-select
-...
-unstruct_event_my_example_event_1:myField::varchar, -- field will be variant type so important to cast
-...
-from 
+SELECT
+    ...
+    unstruct_event_my_example_event_1:myField::varchar, -- field will be variant type so important to cast
+    ...
+FROM
     atomic.events
 ```
 
 </TabItem>
-<TabItem value="databricks" label="Databricks">
+<TabItem value="databricks" label="Databricks, Spark SQL">
 
 Each type of self-describing event is in a dedicated `STRUCT`-type column. The column name and the fields in the `STRUCT` will be determined by the event’s schema. See [how schemas translate to the warehouse](/docs/storing-querying/schemas-in-warehouse/index.md) for more details.
 
 You can query fields in the self-describing event by extracting them like so:
 
 ```sql
-select
-...
-unstruct_event_my_example_event_1.my_field,
-...
-from 
+SELECT
+    ...
+    unstruct_event_my_example_event_1.my_field,
+    ...
+FROM
     atomic.events
 ```
 
+</TabItem>
+<TabItem value="synapse" label="Synapse Analytics 🧪">
+
+Each type of self-describing event is in a dedicated column in JSON format. The column name will be determined by the event’s schema. See [how schemas translate to the warehouse](/docs/storing-querying/schemas-in-warehouse/index.md) for more details.
+
+You can query fields in the self-describing event like so:
+
+```sql
+SELECT
+    ...
+    JSON_VALUE(unstruct_event_my_example_event_1, '$.myField')
+    ...
+FROM
+    OPENROWSET(...) AS [result]
+```
```

Comment:
I might have missed it, but I can't find anywhere we actually give the user an example of how to use `OPENROWSET` or even at least link to the microsoft docs?

Reply:
That will be in the connection instructions on a different page... After all I am probably adding that in this review

Reply:
@rlh1994 I expanded the queries here, and the details on where to get `<events>` are in #611.